### PR TITLE
Cypress/fe 2364 action popover

### PR DESCRIPTION
**Pull request by DlgSHi**
# Description
- added action popover to build classic feature file
- added classic feature file
- added events checking features
- added missing functional tests for Action Popover component

### CODE REVIEW

File: cypress/features/regression/actionPopoverIFrame.feature [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+      | 0     | 1     | Email Invoice |
+      | 1     | 3     | Download PDF  |
+      | 2     | 4     | Download CSV  |
+      | 3     | 6     | Delete        |
+
+  @positive
+  Scenario Outline: Open Action Popover using different keyboard key <key>
+    When I press keyboard "<key>" key times 1 on actionPopover open icon
+    Then Action Popover element "<index>" inner context is set to "<innerText>"
+    Examples:
```
Comment (KatarzynaQA) [inline cypress/features/regression/actionPopoverIFrame.feature]: remove 'index' and 'innerText' from Example if is the same, use it directly in the steps

File: cypress/features/regression/actionPopoverIFrame.feature [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+      | key       | index | innerText     |
+      | enter     | 1     | Email Invoice |
+      | downarrow | 1     | Email Invoice |
+      | space     | 1     | Email Invoice |
+
+  @positive
+  Scenario: Open Action Popover and close it using Home key
```
Comment (KatarzynaQA) [inline cypress/features/regression/actionPopoverIFrame.feature]: Change the name of the scenario, you don't close action popover

File: cypress/features/regression/actionPopoverIFrame.feature [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    When I open Action Popover element
+      And I hit Home on Action Popover element in iFrame
+    Then  Action Popover element "1" inner context is set to "Email Invoice"
+
+  @positive
+  Scenario: Open Action Popover and close it using Uparrow key
```
Comment (KatarzynaQA) [inline cypress/features/regression/actionPopoverIFrame.feature]: Change the name of the scenario, you don't close action popover

File: cypress/features/regression/actionPopoverIFrame.feature [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+    When I open Action Popover element
+      And I hit uparrow on Action Popover element in iFrame
+    Then  Action Popover element "6" inner context is set to "Delete"
+
+  @positive
+  Scenario: Open Action Popover and close it using End key
```
Comment (KatarzynaQA) [inline cypress/features/regression/actionPopoverIFrame.feature]: Change the name of the scenario, you don't close action popover

File: cypress/features/regression/actionPopoverIFrameClassic.feature [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+      | 1     | 3     | Download PDF  |
+      | 2     | 4     | Download CSV  |
+      | 3     | 6     | Delete        |
+
+  @positive
+  Scenario Outline: Open Action Popover using different keyboard key <key>
+    When I press keyboard "<key>" key times 1 on actionPopover open icon
+    Then Action Popover element "<index>" inner context is set to "<innerText>" for classic story
+    Examples:
+      | key       | index | innerText     |
```
Comment (KatarzynaQA) [inline cypress/features/regression/actionPopoverIFrameClassic.feature]: remove 'index' and 'innerText' from Example if is the same, use it directly in the steps

File: cypress/features/regression/actionPopoverIFrameClassic.feature [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+      | enter     | 1     | Email Invoice |
+      | downarrow | 1     | Email Invoice |
+      | space     | 1     | Email Invoice |
+
+  @positive
+  Scenario: Open Action Popover and close it using Home key
```
Comment (KatarzynaQA) [inline cypress/features/regression/actionPopoverIFrameClassic.feature]: Change the name of the scenario, you don't close action popover

File: cypress/features/regression/actionPopoverIFrameClassic.feature [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+    When I open Action Popover element
+      And I hit Home on Action Popover element in iFrame
+    Then  Action Popover element "1" inner context is set to "Email Invoice" for classic story
+
+  @positive
+  Scenario: Open Action Popover and close it using uparrow key
```
Comment (KatarzynaQA) [inline cypress/features/regression/actionPopoverIFrameClassic.feature]: Change the name of the scenario, you don't close action popover

File: cypress/features/regression/actionPopoverIFrameClassic.feature [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+    When I open Action Popover element
+      And I hit uparrow on Action Popover element in iFrame
+    Then  Action Popover element "6" inner context is set to "Delete" for classic story
+
+  @positive
+  Scenario: Open Action Popover and close it using End key
```
Comment (KatarzynaQA) [inline cypress/features/regression/actionPopoverIFrameClassic.feature]: Change the name of the scenario, you don't close action popover

File: cypress/features/regression/actionPopoverEvents.feature [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+    Then csv action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover element, click on it and check delete event
+    When clear all actions in Actions Tab
+      And I click 6 actionPopoverInnerItem
+    Then delete action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check email event for actions after pressing enter key
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEvents.feature]: fixed

File: cypress/features/regression/actionPopoverEvents.feature [hunk 10]
```diff
@@ -0,0 +1,54 @@
...
+    When clear all actions in Actions Tab
+      And I press "enter" actionPopoverInnerItem onto 1 element
+    Then email action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check pdf event for actions after pressing enter key
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEvents.feature]: fixed

File: cypress/features/regression/actionPopoverEvents.feature [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+    When clear all actions in Actions Tab
+      And I press "enter" actionPopoverInnerItem onto 3 element
+    Then pdf action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check csv event for actions after pressing enter key
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEvents.feature]: fixed

File: cypress/features/regression/actionPopoverEvents.feature [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+    When clear all actions in Actions Tab
+      And I press "enter" actionPopoverInnerItem onto 4 element
+    Then csv action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check delete event for actions after pressing enter key
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEvents.feature]: fixed

File: cypress/features/regression/actionPopoverEventsClassic.feature [hunk 13]
```diff
@@ -0,0 +1,54 @@
...
+    Then pdf action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check csv event for actions after pressing enter key
+    When clear all actions in Actions Tab
+      And I press "enter" actionPopoverInnerItem onto 4 element
+    Then csv action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check delete event for actions after pressing enter key
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEventsClassic.feature]: fixed

File: cypress/features/regression/actionPopoverEventsClassic.feature [hunk 14]
```diff
@@ -0,0 +1,54 @@
...
+    Then delete action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check email event for actions after pressing enter key
+    When clear all actions in Actions Tab
+      And I press "enter" actionPopoverInnerItem onto 1 element
+    Then email action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check pdf event for actions after pressing enter key
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEventsClassic.feature]: fixed

File: cypress/features/regression/actionPopoverEventsClassic.feature [hunk 15]
```diff
@@ -0,0 +1,54 @@
...
+    Then csv action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover element, click on it and check delete event
+    When clear all actions in Actions Tab
+      And I click 6 actionPopoverInnerItem
+    Then delete action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check email event for actions after pressing enter key
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEventsClassic.feature]: fixed

File: cypress/features/regression/actionPopoverEventsClassic.feature [hunk 16]
```diff
@@ -0,0 +1,54 @@
+Feature: Action Popover component
+  I want to change Alert component properties
+
+  Background: Open Action Popover component page
+    Given I open "Action Popover" component page classic
+      And I open Action Popover element page with iFrame
+
+  @positive
+  Scenario: Open Action Popover element, click on it and check email event
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEventsClassic.feature]: fixed

File: cypress/features/regression/actionPopoverEvents.feature [hunk 17]
```diff
@@ -0,0 +1,54 @@
+Feature: Action Popover component
+  I want to change Alert component properties
+
+  Background: Open Action Popover component page
+    Given I open "Action Popover" component page
+      And I open Action Popover element page with iFrame
+
+  @positive
+  Scenario: Open Action Popover element, click on it and check email event
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEvents.feature]: fixed

File: cypress/features/regression/actionPopoverEvents.feature [hunk 18]
```diff
@@ -0,0 +1,54 @@
...
+      And I click 1 actionPopoverInnerItem
+    Then email action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover element, click on it and check pdf event
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEvents.feature]: fixed

File: cypress/features/regression/actionPopoverEvents.feature [hunk 19]
```diff
@@ -0,0 +1,54 @@
...
+      And I click 3 actionPopoverInnerItem
+    Then pdf action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover element, click on it and check csv event
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEvents.feature]: fixed

File: cypress/features/regression/actionPopoverEvents.feature [hunk 20]
```diff
@@ -0,0 +1,54 @@
...
+      And I click 4 actionPopoverInnerItem
+    Then csv action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover element, click on it and check delete event
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEvents.feature]: fixed

File: cypress/features/regression/actionPopoverEvents.feature [hunk 21]
```diff
@@ -0,0 +1,54 @@
...
+      And I click 6 actionPopoverInnerItem
+    Then delete action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check email event for actions after pressing enter key
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEvents.feature]: fixed

File: cypress/features/regression/actionPopoverEvents.feature [hunk 22]
```diff
@@ -0,0 +1,54 @@
...
+      And I press "enter" actionPopoverInnerItem onto 1 element
+    Then email action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check pdf event for actions after pressing enter key
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEvents.feature]: fixed

File: cypress/features/regression/actionPopoverEvents.feature [hunk 23]
```diff
@@ -0,0 +1,54 @@
...
+      And I press "enter" actionPopoverInnerItem onto 3 element
+    Then pdf action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check csv event for actions after pressing enter key
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEvents.feature]: fixed

File: cypress/features/regression/actionPopoverEvents.feature [hunk 24]
```diff
@@ -0,0 +1,54 @@
...
+      And I press "enter" actionPopoverInnerItem onto 4 element
+    Then csv action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check delete event for actions after pressing enter key
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEvents.feature]: fixed

File: cypress/features/regression/actionPopoverEventsClassic.feature [hunk 25]
```diff
@@ -0,0 +1,54 @@
...
+      And I click 1 actionPopoverInnerItem
+    Then email action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover element, click on it and check pdf event
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEventsClassic.feature]: fixed

File: cypress/features/regression/actionPopoverEventsClassic.feature [hunk 26]
```diff
@@ -0,0 +1,54 @@
...
+      And I click 3 actionPopoverInnerItem
+    Then pdf action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover element, click on it and check csv event
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEventsClassic.feature]: fixed

File: cypress/features/regression/actionPopoverEventsClassic.feature [hunk 27]
```diff
@@ -0,0 +1,54 @@
...
+      And I click 4 actionPopoverInnerItem
+    Then csv action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover element, click on it and check delete event
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEventsClassic.feature]: fixed

File: cypress/features/regression/actionPopoverEventsClassic.feature [hunk 28]
```diff
@@ -0,0 +1,54 @@
...
+      And I click 6 actionPopoverInnerItem
+    Then delete action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check email event for actions after pressing enter key
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEventsClassic.feature]: fixed

File: cypress/features/regression/actionPopoverEventsClassic.feature [hunk 29]
```diff
@@ -0,0 +1,54 @@
...
+      And I press "enter" actionPopoverInnerItem onto 1 element
+    Then email action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check pdf event for actions after pressing enter key
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEventsClassic.feature]: fixed

File: cypress/features/regression/actionPopoverEventsClassic.feature [hunk 30]
```diff
@@ -0,0 +1,54 @@
...
+      And I press "enter" actionPopoverInnerItem onto 3 element
+    Then pdf action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check csv event for actions after pressing enter key
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEventsClassic.feature]: fixed

File: cypress/features/regression/actionPopoverEventsClassic.feature [hunk 31]
```diff
@@ -0,0 +1,54 @@
...
+    Then email action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check pdf event for actions after pressing enter key
+    When clear all actions in Actions Tab
+      And I press "enter" actionPopoverInnerItem onto 3 element
+    Then pdf action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check csv event for actions after pressing enter key
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEventsClassic.feature]: fixed

File: cypress/features/regression/actionPopoverEventsClassic.feature [hunk 32]
```diff
@@ -0,0 +1,54 @@
...
+    When clear all actions in Actions Tab
+      And I press "enter" actionPopoverInnerItem onto 4 element
+    Then csv action was called in Actions Tab
+
+  @positive
+  Scenario: Open Action Popover, press downarrow key and check delete event for actions after pressing enter key
+    When clear all actions in Actions Tab
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEventsClassic.feature]: fixed

File: cypress/features/regression/actionPopoverIFrame.feature (hunk 1 above)
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverIFrame.feature]: fixed

File: cypress/features/regression/actionPopoverIFrame.feature (hunk 2 above)
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverIFrame.feature]: fixed

File: cypress/features/regression/actionPopoverIFrame.feature (hunk 3 above)
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverIFrame.feature]: fixed

File: cypress/features/regression/actionPopoverIFrame.feature (hunk 4 above)
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverIFrame.feature]: fixed

File: cypress/features/regression/actionPopoverIFrameClassic.feature (hunk 5 above)
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverIFrameClassic.feature]: fixed

File: cypress/features/regression/actionPopoverIFrameClassic.feature (hunk 6 above)
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverIFrameClassic.feature]: fixed

File: cypress/features/regression/actionPopoverIFrameClassic.feature (hunk 7 above)
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverIFrameClassic.feature]: fixed

File: cypress/features/regression/actionPopoverIFrameClassic.feature (hunk 8 above)
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverIFrameClassic.feature]: fixed

File: cypress/features/regression/actionPopoverIFrame.feature [hunk 33]
```diff
@@ -0,0 +1,94 @@
...
+    Given I open Action Popover element
+    When I open Action Popover element
+    Then Action Popover element is not visible
+
+  @positive
+  Scenario Outline: Open Action Popover and focus <innerText> element using different keyboard key <key>
+    Given I open Action Popover element
+    When I press keyboard "<key>" key times <times>
+    Then Action Popover element is visible
+      And Action Popover element "<index>" inner context is set to "<innerText>"
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverIFrame.feature]: fixed

File: cypress/features/regression/actionPopoverIFrameClassic.feature [hunk 34]
```diff
@@ -0,0 +1,94 @@
+Feature: Action Popover component classic
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverIFrameClassic.feature]: removed, only border-color tests left

File: cypress/features/regression/actionPopoverEventsClassic.feature [hunk 35]
```diff
@@ -0,0 +1,47 @@
+Feature: Action Popover component
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverEventsClassic.feature]: deleted

File: cypress/features/regression/actionPopoverIFrame.feature [hunk 36]
```diff
@@ -0,0 +1,94 @@
+Feature: Action Popover component
+  I want to change Alert component properties
+
+  Background: Open Action Popover component page
+    Given I open "Action Popover" component iframe
+
+  @positive
+  Scenario: Open Action Popover element
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverIFrame.feature]: those tests are below in file

File: cypress/support/step-definitions/action-popover-steps.js [hunk 37]
```diff
@@ -1,24 +1,80 @@
...
+    actionPopoverButton().first().type(`${key}`);
+  }
+});
+
+Then('Action Popover element {word} inner context is set to {string}', (element, text) => {
+  cy.focused().should('contain', text);
+  cy.focused().should('have.css', 'box-shadow', 'rgb(255, 181, 0) 0px 0px 0px 2px inset');
+});
+
+Then('Action Popover element {word} inner context is set to {string} for classic story', (element, text) => {
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/support/step-definitions/action-popover-steps.js]: fixed

File: cypress/support/step-definitions/action-popover-steps.js [hunk 38]
```diff
@@ -1,24 +1,80 @@
...
-Then('Action Popover element {string} inner context is set to {string}', (element, text) => {
-  cy.focused().should('have', actionPopoverInnerItem(element));
-  actionPopoverInnerItem(element).invoke('text').should('contain', text);
+When('I press keyboard {string} key times {int} on actionPopover open icon', (key, times) => {
+  for (let i = 0; i < times; i++) {
+    actionPopoverButton().first().type(`${key}`);
+  }
+});
+
+Then('Action Popover element {word} inner context is set to {string}', (element, text) => {
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/support/step-definitions/action-popover-steps.js]: fixed

File: cypress/features/regression/actionPopoverIFrame.feature [hunk 39]
```diff
@@ -0,0 +1,94 @@
...
+
+  @positive
+  Scenario: Open Action Popover and close it by clicking outside of the component
+    Given I open Action Popover element
+    When I click on outside dialog in iFrame
+    Then Action Popover element is not visible
+
+  @positive
+  Scenario: Open Action Popover and close it by clicking onto Open icon
+    Given I open Action Popover element
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverIFrame.feature]: fixed

File: cypress/features/regression/actionPopoverIFrame.feature [hunk 40]
```diff
@@ -0,0 +1,94 @@
...
+      | key       |
+      | enter     |
+      | downarrow |
+      | space     |
+
+  @positive
+  Scenario: Open Action Popover and verify that the first element Email Invoice is focused using Home key
+    When I open Action Popover element
+      And I hit Home on Action Popover element in iFrame
+    Then  Action Popover element "1" inner context is set to "Email Invoice"
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverIFrame.feature]: fixed

File: cypress/support/step-definitions/action-popover-steps.js [hunk 41]
```diff
@@ -1,24 +1,80 @@
...
+When('I open Action Popover element page with iFrame', () => {
+  actionPopoverButtonNoIframe().eq(0).click();
+});
+
+When('I hit ESC on Action Popover element in iFrame', () => {
+  cy.focused().trigger('keydown', { keyCode: 16, which: 16, release: false });
+  cy.focused().trigger('keydown', { keyCode: 27, which: 27 });
+});
+
+When('I hit Tab on Action Popover element in iFrame', () => {
```
[the remark this answers is not in the document]
Comment (DlgSHi) [reply cypress/support/step-definitions/action-popover-steps.js]: changed and moved those steps to `common-steps.js`

File: cypress/features/regression/actionPopoverIFrameClassic.feature [hunk 42]
```diff
@@ -0,0 +1,18 @@
...
+    Given I open "Action Popover" component for classic story in iframe
+
+  @positive
+  Scenario Outline: Open Action Popover element and check <innerText> as inner context
+    When I click the menu button element
+      And I press keyboard "downarrow" key times <times>
+    Then focused element inner content is set to "<innerText>"
+      And Action Popover element has blue border on focus
+    Examples:
+      | times | index | innerText     |
```
Comment (jamime) [inline cypress/features/regression/actionPopoverIFrameClassic.feature]: `index` is not used

File: cypress/support/step-definitions/common-steps.js [hunk 43]
```diff
@@ -397,3 +397,43 @@ Then('{string} tab in {string} tab list is visible', (knobsName, position) => {
     default: throw new Error('There are only two tab list elements on the page');
   }
 });
+
+When('I press ESC in iFrame', () => {
+  cy.focused().trigger('keydown', { keyCode: 16, which: 16, release: false });
+  cy.focused().trigger('keydown', { keyCode: 27, which: 27 });
+});
+
+When('I press Tab in iFrame', () => {
```
Comment (jamime) [inline cypress/support/step-definitions/common-steps.js]: These steps have no reference to iframe, please update them `I press Tab` could be used in many places without iframe.

Comment (jamime) [reply cypress/support/step-definitions/common-steps.js]: Same with the other steps in this file

File: cypress/features/regression/actionPopoverIFrameClassic.feature (hunk 42 above)
Comment (DlgSHi) [reply cypress/features/regression/actionPopoverIFrameClassic.feature]: fixed

File: cypress/support/step-definitions/common-steps.js (hunk 43 above)
Comment (DlgSHi) [reply cypress/support/step-definitions/common-steps.js]: agree, fixed

**Pull request merged**